sdk/servicebus/service-bus/src/models.ts

```diff
@@ -3,6 +3,35 @@
 
 import { OperationOptionsBase } from "./modelsToBeSharedWithEventHubs";
 import Long from "long";
+import { MessagingError } from "@azure/core-amqp";
+
+/**
+ * Arguments to the `processError` callback.
+ */
+export interface ProcessErrorArgs {
+  /**
+   * The error.
+   */
+  error: Error | MessagingError;
+  /**
+   * The operation where the error originated.
+   *
+   * 'abandon': Errors that occur when if `abandon` is triggered automatically.
```

Comment:
```suggestion
   * 'abandon': Errors that occur in the process of abandoning a message by the receiver as a reaction to error from the user's `processMessage` callback or error from completing the message as per the `autoComplete` feature.
```

Reply:
Let's leave this one as it is (unless you can come up with something shorter). My bigger concern at this point is that the help text is already going to be long because I had to shove _all_ the descriptions onto errorSource's JSDoc.

Also, I think it adequately describes when this gets called. We can describe the various spots that abandon can happen in subscribe(), perhaps?